src/actions/index.js

```diff
@@ -3,12 +3,12 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // @flow
-import * as app from './app';
```

Comment:
The changes in `src/action/index.js` can be reverted. They are actually breaking things so that the test aren't passing. I believe we updated the issue to allow for relative paths when they are in the same directory.

Reply:
I see. So I should replace absolute with relative paths only for files that are in actions directory?